### ISSUE TITLE
tf load model is erro

### ISSUE THREAD

**Issue by jangjiun** [rishigami/Swin-Transformer-TF#5]
import tensorflow as tf
from swintransformer import SwinTransformer
model = tf.keras.Sequential([
  tf.keras.layers.Lambda(lambda data: tf.keras.applications.imagenet_utils.preprocess_input(tf.cast(data, tf.float32), mode="torch"), input_shape=[*IMAGE_SIZE, 3]), SwinTransformer('swin_tiny_224', include_top=False, pretrained=True),
  tf.keras.layers.Dense(NUM_CLASSES, activation='softmax')
])

tf can't load pre trained  model。this step is errro
